### PR TITLE
Complete re-enabling github action

### PR DESCRIPTION
**Pull request by 0xcodercrane**
Resolves #https://github.com/ubiquity/ubiquibot/issues/310

### CODE REVIEW

**Comment by 0xcodercrane** [comment]
[here](https://github.com/0xcodercrane/ubiquibot/issues/7)'s the QA for this

**Comment by 0xcodercrane** [comment]
The problem with `tsx adapters/index.ts` is that it requires `yarn install` and `yarn build`. this slow down the execution time of the action. but i leave it for now bc i guess its the original spec to use `tsx adapters/index.ts` @pavlovcik

**Comment by pavlovcik** [comment]
> The problem with `tsx adapters/index.ts` is that it requires `yarn install` and `yarn build`. this slow down the execution time of the action. but i leave it for now bc i guess its the original spec to use `tsx adapters/index.ts` @pavlovcik 

`yes | npx tsx adapters/index.ts`

`yarn build` it's inconvenient for me to check from my phone now what it does but typically this just compiles typescript into javascript. This is not a necessary step when using the `tsx` runtime

**Comment by 0xcodercrane** [comment]
you're right. we can remove `yarn build`

File: src/handlers/push/index.ts [hunk 1]
```diff
@@ -26,6 +26,6 @@ export const runOnPush = async () => {
   // check for modified or added files and check for specified file
   if (payload.head_commit.modified.includes(BASE_RATE_FILE) || payload.head_commit.added.includes(BASE_RATE_FILE)) {
     // update base rate
-    updateBaseRate(context, payload, BASE_RATE_FILE);
+    await updateBaseRate(context, payload, BASE_RATE_FILE);
```
Comment (pavlovcik) [inline src/handlers/push/index.ts:29]: I'm pretty sure I fixed this in the original PR. How was it removed?

Comment (0xcodercrane) [reply src/handlers/push/index.ts:29]: when I catch it up, it was not there. so I think I added it again.

Comment (pavlovcik) [reply src/handlers/push/index.ts:29]: I made a mistake and realized that it is a different function with a similar name that I fixed in https://github.com/ubiquity/ubiquibot/pull/352#discussion_r1216587731

Comment (0xcodercrane) [reply src/handlers/push/index.ts:29]: Yeah that was the only part but I can go through it.

Comment (0xcodercrane) [reply src/handlers/push/index.ts:29]: I don't think we should fix `await` problems in this PR. I suggest to do that out of this PR.

**Review by pavlovcik — DISMISSED** [review]

**Comment by pavlovcik** [comment]
CI isn't passing.

**Pull request merged**